docs/pywbemcligeneraloptions.rst

```diff
@@ -61,166 +70,470 @@ command, general the options redefinition's are not retained between command
+timeout (normally 30 seconds) if this option is not defined.
+
+
+.. _`--no-verify general option`:
+
+--no-verify general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the ``--no-verify/-n`` boolean general options is set, theclient does not
+verify any certificates received from the WBEM server. Any certificate returned
+by the WBEM server is accepted.
+
+
+.. _`--certfile general option`:
+
+--certfile general option
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--certfile`` is a file path for a X.509 client certificate
+to be presented to the WBEM server along with the ``--keyfile`` option during
+the TLS/SSL handshake. This parameter used only with HTTPS.  If None, no client
+certificate is presented to the server, enabling 1-way authentication.
+Otherwise, the client certificate is presented to the server, enabling 2-way
+(mutual) authentication. This and the ``--keyfile`` are presented to the pywbem
+:class:`pywbem.WBEMConnection` as a single parameter ``X-509``:
+
+* ``--cert_file`` - The file path of a file containing an X.509 client
+  certificate.
+* ``key_file`` - The file path of a file containing the private key belonging
+  to the public key that is part of the X.509 certificate file.
+
+For more information on authentication types, see:
+:https://pywbem.readthedocs.io/en/stable/client/security.html#authentication-types
+
+.. _`--keyfile general option`:
+
+----keyfile general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--keyfile`` general option is a file path of the
+the client private key file containing the private key belonging to the public
+key that is part of the X.509 certificate. See ``--certfile`` for more
+information. Not required if the private key is part of the file defined in
+``--certfile``. Not allowed if ``--certfile`` option not provided. Default: No
+client key file. Client private key should then be part of the file defined by
+``--certfile``.
+
+The pywbemcli command line options are as follows (The following complements
+the exact help text output shown in section :ref:`pywbemcli Help Command
+Details`)
+
+
+.. _`--ca-certs general option`:
+
+----ca-certs general option
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--ca-certs`` a file or directory containing certificates
+that will be matched against acertificate received from WBEM server. Setting
+the :ref:`--no-verify general option` bypasses client verification of the
+WBEM server certificate. The default is OS dependent and is a set of system
+directories where certficates are expected to be stored for the client OS.
+
+
+.. _`--use-pull general option`:
+
+--use_pull general option
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The value of the ``--use-pull/-u`` general option is one of the following
+strings: [``yes``|``no``|``either``] that determines whether the pull
+operations are used for ``EnumerateInstances``, ``AssociatorInstances``,
+``ReferenceInstances``, and ``ExecQuery`` operations See :ref:`Pywbemcli and
```

Comment:
These are the names of the traditional CIM-XML operations, so pull operations are never used for them.

I suggest to name the pywbemcli commands instead of mentioning pywbem methods, i.e. replace the list above with "are used for the instance command group".

Reply:
Looks better that way.